### PR TITLE
Parent #738: Fix CSS issues in development

### PR DESCRIPTION
**Pull request by simsausaurabh**
**Changes proposed in this pull request**
- Made changes to fix UI bugs in development.
- Made changes to support host listener decorator.
- Fixes part of parent issue - #738
**Screenshots (if appropriate)** 

**Link to live demo:http://pr-740-fossasia-loklaksearch.surge.sh** 

**Closes #**

### CODE REVIEW

**Comment by simsausaurabh** [comment]
@hemantjadon please review.

I think development branch needs to enabled in project settings since its not getting analyzed by Codacy?

screenshot from 2018-06-03 19-25-31

**Comment by simsausaurabh** [comment]
Before:
screenshot from 2018-06-03 19-32-39

After:
screenshot from 2018-06-03 19-31-07

I am working on other UI issues and broken search results.

File: tslint.json [hunk 1]
```diff
@@ -119,7 +119,7 @@
     "no-output-on-prefix": true,
     "use-input-property-decorator": true,
     "use-output-property-decorator": true,
-    "use-host-property-decorator": true,
+    "use-host-property-decorator": false,
```
Comment (hemantjadon) [inline tslint.json]: I think this should not be turned off.

Comment (simsausaurabh) [reply tslint.json]: Oh, sorry I was turning it to true, didn't realise it was already by taking reference from earlier branch.

**Comment by simsausaurabh** [comment]
Oh, sorry I was turning it to true, didn't realise it was already by taking reference from earlier branch. I have updated the changes please review. @hemantjadon

**Pull request merged**